### DISCUSSION TITLE
AttributeError: 'Message' object has no attribute 'reply'

### DISCUSSION THREAD

**Discussion by Manhunter7** [Rapptz/discord.py#6123] [Q&A]
Hello, when I run my program it gives me this error. It looks like the compiler doesn't want to take the .reply command.
I code with PyCharm and i've installed discord.py properly
P.S. : Sorry if you see this message twice :)

**Comment by ghost** [comment]


**Reply by Harmon758** [reply]
`Message.reply` exists as a method in the latest development version on the master branch.

**Comment by Harmon758** [comment]
Currently, the latest released version of discord.py, v1.5.1, does not include support for inline replies.
`Message.reply` has been implemented in the latest development version on the master branch.
